lib/galaxy/jobs/runners/util/cli/shell/rsh.py

```diff
@@ -37,15 +40,13 @@ class SecureShell(RemoteShell):
 
     def __init__(self, rsh='ssh', rcp='scp', private_key=None, port=None, strict_host_key_checking=True, **kwargs):
         strict_host_key_checking = "yes" if strict_host_key_checking else "no"
-        rsh += " -oStrictHostKeyChecking=%s -oConnectTimeout=60" % strict_host_key_checking
-        rcp += " -oStrictHostKeyChecking=%s -oConnectTimeout=60" % strict_host_key_checking
+        options = ["-oStrictHostKeyChecking=%s" % strict_host_key_checking]
+        options.append("-oConnectTimeout=60")
```

Comment:
I think that `-o Keyword=Value` is more common, what about:
```python
        options = ["-o", "StrictHostKeyChecking=%s" % strict_host_key_checking]
        options.extend(["-o", "ConnectTimeout=60"])
```

Reply:
sure, looks better